tool/resources/org/antlr/v4/tool/templates/codegen/Python3/Python3.stg

```diff
@@ -652,6 +652,7 @@ CaptureNextTokenType(d) ::= "<d.varName> = self._input.LA(1)"
 
 StructDecl(struct,ctorAttrs,attrs,getters,dispatchMethods,interfaces,extensionMembers) ::= <<
 class <struct.name>(<if(contextSuperClass)><contextSuperClass><else>ParserRuleContext<endif>):
+    __slots__ = ('parser', '__dict__')
```

Comment:
Adding `__dict__` here as a catch-all for any other class members (labeled grammar rule members?)
Ideally these would be emitted here too, but I'm not familiar enough with how the parser generator templates work to implement this, so using `__dict__` is sufficient.

Reply:
this should be as simple as:
__slots__ = ('parser', <attrs:{a | '<a>'}; separator=",">) 
can you try ?then we can leverage your perf improvement measures